### ISSUE TITLE
Avoid possible naming confusion - use of community

### ISSUE THREAD

**Issue by svx** [plone/ploneorg.core#169]
Currently the use of  the word *community* on plone.org is highly confusing at least for me.

In the top header we have a link to community.plone.org this if fine, great, but than in the second bar, we have it again, linking to https://plone.org/community, which is different than the first one.

Here you get:

![po-confuse](https://cloud.githubusercontent.com/assets/358860/15567954/b183a50a-2329-11e6-8149-a8b3a220b001.png)

Which let me think I can sign here up for community.plone.org, but no, it links to https://plone.org/community/sign-me-up

This is kind of confusing are we talking different communities ?

**Comment by tkimnguyen** [comment]
Yeah, the site community.plone.org should maybe have been named forum.plone.org... but I don't know if there would be support for changing the subdomain. In the top header we *could* say "Forum" instead of "community.plone.org". What do you think? 
![image](https://cloud.githubusercontent.com/assets/815418/15609840/dbfc0bd8-23e7-11e6-84f2-87989f418aa8.png)

**Comment by djay** [comment]
+1

**Comment by davilima6** [comment]
It'd be interesting to setup HTTP 301s so we don't completely lose Google indexing.
